chapters/data_processing/transform_images.ipynb

```diff
@@ -0,0 +1,4003 @@
+      ],
+      "text/plain": [
+       "ExecutionResult(\n",
+       "    component_id: ImportExampleGen\n",
+       "    execution_id: 1\n",
+       "    outputs:\n",
+       "        examples: Channel(\n",
+       "            type_name: Examples\n",
+       "            artifacts: [Artifact(artifact: id: 1\n",
+       "        type_id: 5\n",
+       "        uri: \"./flowers_tfx/ImportExampleGen/examples/1\"\n",
+       "        properties {\n",
+       "          key: \"split_names\"\n",
+       "          value {\n",
+       "            string_value: \"[\\\"train\\\", \\\"test\\\"]\"\n",
+       "          }\n",
+       "        }\n",
+       "        custom_properties {\n",
+       "          key: \"input_fingerprint\"\n",
+       "          value {\n",
+       "            string_value: \"split:single_split,num_files:1,total_bytes:1100898,xor_checksum:1613383800,sum_checksum:1613383800\"\n",
+       "          }\n",
+       "        }\n",
+       "        custom_properties {\n",
+       "          key: \"payload_format\"\n",
+       "          value {\n",
+       "            string_value: \"FORMAT_TF_EXAMPLE\"\n",
+       "          }\n",
+       "        }\n",
+       "        custom_properties {\n",
+       "          key: \"span\"\n",
+       "          value {\n",
+       "            string_value: \"0\"\n",
+       "          }\n",
+       "        }\n",
+       "        custom_properties {\n",
+       "          key: \"state\"\n",
+       "          value {\n",
+       "            string_value: \"published\"\n",
+       "          }\n",
+       "        }\n",
+       "        state: LIVE\n",
+       "        , artifact_type: id: 5\n",
+       "        name: \"Examples\"\n",
+       "        properties {\n",
+       "          key: \"span\"\n",
+       "          value: INT\n",
+       "        }\n",
+       "        properties {\n",
+       "          key: \"split_names\"\n",
+       "          value: STRING\n",
+       "        }\n",
+       "        properties {\n",
+       "          key: \"version\"\n",
+       "          value: INT\n",
+       "        }\n",
+       "        )]\n",
+       "            additional_properties: {}\n",
+       "            additional_custom_properties: {}\n",
+       "        ))"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "example_gen = ImportExampleGen(input_base='./data',\n",
+    "                               output_config=output)\n",
+    " \n",
+    "context.run(example_gen, enable_cache=False)"
```

Comment:
Is there a specific reason why you need to disable the cache here? If so, please leave a comment.

Reply:
Not really. Still experimenting :-)